frontend/src/components/VSearchResultsGrid/VImageGrid.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section>
     <VGridSkeleton
-      v-if="!images.length && !fetchState.isFinished"
+      v-if="images != null && !images.length && !fetchState.isFinished"
```

Comment:
This also accounts for undefined.
```suggestion
      v-if="images && !images.length && !fetchState.isFinished"
```